src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -83,7 +84,9 @@ export default async function TaskDetailPage({
 
   console.info(`app/detail/${task_id}/${user_type}/page.tsx | Serving user ${token} with payload`, tokenPayload)
 
-  redirectIfResourceNotFound(searchParams, task, !!tokenPayload.internalUserId)
+  if (!!!task) {
```

Comment:
Wouldn't a single `!` be enough here?

Reply:
Fixed in the commit 26a2a7b244909e7c7467cbea75c4ba08250d0dcf